object/syncer.go

```diff
@@ -41,7 +41,7 @@ type Syncer struct {
 	Port             int            `json:"port"`
 	User             string         `xorm:"varchar(100)" json:"user"`
 	Password         string         `xorm:"varchar(100)" json:"password"`
-	DatabaseType     string         `xorm:"varchar(100)" json:"databaseType"`
+	DriverName       string         `xorm:"varchar(100)" json:"driverName"`
```

Comment:
Rename `DatabaseType` to `DriverName`, field of pages in frontend, swagger, and i18n updated.